### PR TITLE
Open orders table

### PR DESCRIPTION
**Pull request by phoebemirman**
fixes https://github.com/AugurProject/augur/issues/1079

- added view switching table component with searching
   - QuadBox is where the appearance of the quads lives and other components will manage its functionality
- implemented open orders and filled orders to use it

### CODE REVIEW

**Comment by AugurIntegration** [comment]
This pull request **introduces 13 alerts** when merging c862bc23ba9046378b219e018218b48494a97cfa into c13cb41b73ad9b8efbb9ced3e925d19c921a3eb9 - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur-ui/rev/pr-d063d7e539e8d3663df2f5ad984fa09b6950cbad)

**new alerts:**

* 10 for Unused variable, import, function or class
* 2 for Potentially inconsistent state update
* 1 for Direct state mutation

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by andrewdamelio** [comment]
LGTM 👍 nice job setting up the containers for this page.

This PR provided me a lot of codebase learnings - thanks @phoebemirman

File: src/modules/portfolio/components/orders/filled-orders.jsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    this.filterComp = this.filterComp.bind(this);
+    this.switchView = this.switchView.bind(this);
+  }
+
+  componentWillMount() {
+    const { loadAccountTrades } = this.props;
+    loadAccountTrades();
+  }
+
+  updateFilteredData(filteredData) {
```
Comment (bthaile) [inline src/modules/portfolio/components/orders/filled-orders.jsx]: all these methods could be arrow functions then you don't have to bind to `this`

File: src/modules/portfolio/containers/filled-orders.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    }
+  });
+
+  return {
+    currentTimestamp: selectCurrentTimestamp(state),
+    marketsCount,
+    transactionsStatus: state.transactionsStatus,
+    markets,
+    transactionsLoading: state.appStatus.transactionsLoading,
+    registerBlockNumber: state.loginAccount.registerBlockNumber,
```
Comment (bthaile) [inline src/modules/portfolio/containers/filled-orders.js]: What is `registerBlockNumber` ?

**Pull request merged**